### PR TITLE
Document customQueryParameters for prometheus datasource provisioning

### PR DESCRIPTION
**Pull request by aecay**
**What this PR does / why we need it**:

Document customQueryParameters for prometheus datasource provisioning

The customQueryParameters option was added to prometheus datasources in https://github.com/grafana/grafana/pull/19121.  This PR updates the "provisioning" documentation to contain a reference to this in the `jsonData` options for a prometheus datasource.

**Which issue(s) this PR fixes**:

none afaik -- just corrects a gap in the docs

### CODE REVIEW

**Comment by achatterjee-grafana** [comment]
Need developer review.

**Review by achatterjee-grafana — APPROVED** [review]

File: docs/sources/administration/provisioning.md [hunk 1]
```diff
@@ -153,6 +153,7 @@ Since not all datasources have the same configuration settings we only have the
 | httpMode                | string  | Influxdb                                                         | HTTP Method. 'GET', 'POST', defaults to GET                                                 |
 | maxSeries               | number  | Influxdb                                                         | Max number of series/tables that Grafana processes                                          |
 | httpMethod              | string  | Prometheus                                                       | HTTP Method. 'GET', 'POST', defaults to GET                                                 |
+| customQueryParameters   | string  | Prometheus                                                       | Query parameters to add, as a url-encoded string.                                                 |
```
Comment (achatterjee-grafana) [inline docs/sources/administration/provisioning.md:156]:
```suggestion
| customQueryParameters   | string  | Prometheus                                                       | Query parameters to add, as a URL-encoded string.                                                 |
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Change url to URL

**Comment by achatterjee-grafana** [comment]
@aecay, you need to sign the CLA agreement. Thanks!

**Review by ivanahuckova — APPROVED** [review]
Looks good! Thanks for adding this @aecay!🙏 @achatterjee-grafana if you have no more comments and all looks good, I think that we can merge it. :)

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Comment by ivanahuckova** [comment]
There seems to be a problem with failing CI. I have restarted the CI, which might fix it.  But if not @aecay maybe you'll need to merge master.

**Pull request merged**